custom_components/windcentrale/const.py

```diff
@@ -78,4 +79,4 @@ def __init__(self, windturbine_name, windturbine_code, windturbine_shares):
         self.shares = windturbine_shares
     
     def toJSON(self):
-        return json.dumps(self, default=lambda o: o.__dict__, sort_keys=True, indent=4)
\ No newline at end of file
+        return json.dumps(self, default=lambda o: o.__dict__, sort_keys=True, indent=4)
```

Comment:
Nothing changed?

Reply:
No, i think i removed a new-line character by accident (but not bad either)